rust-toolchain.toml

```diff
@@ -4,6 +4,6 @@
 [toolchain]
 # Writing custom lints requires using nightly Rust. We pin to a specific version of nightly version
 # so that builds are reproducible.
-channel = "nightly-2024-11-14"
+channel = "nightly-2024-11-28"
```

Comment:
Maybe write a script, that checks that this version, the one in the README and the `clippy_utils` version match. Should be possible with some `cargo metadata` magic and pulling from crates.io. Something like this:

```
VERSION=$(cargo metadata --format-version 1 | jq '.packages[] | select(.name == "clippy_utils")' | jq -r '.version')
EXP_NIGHTLY=$(wget -qO- https://crates.io/api/v1/crates/clippy_utils/${VERSION}/readme | grep -Po "nightly-\d\d\d\d-\d\d-\d\d")

RTC_NIGHTLY=$(grep -Po "nightly-\d\d\d\d-\d\d-\d\d" rust-toolchain.toml)
RM_NIGHTLY=$(grep -Po "nightly-\d\d\d\d-\d\d-\d\d" bevy_lint/README.md | head -1)

test $EXP_NIGHTLY == $RTC_NIGHTLY
test $EXP_NIGHTLY == $RM_NIGHTLY
```

The `RM_NIGHTLY` command needs to be refined, as there are other nightly mentions in the file. The rest should work with maybe some minor adjustments.

Reply:
Good idea! I'm going to spin that out into a follow-up PR :)